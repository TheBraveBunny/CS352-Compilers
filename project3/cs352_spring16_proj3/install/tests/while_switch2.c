int thing;

int main()
{
   int i = 10;
   while(i>=10){
	   switch(i){
		case 10:
			printf("1!\n");
			i=i*2;
			continue;
		case 27:
		case 28:
		case 29:
		case 30:
			i=i/i;
			break;
		default: 
			printf("not 1!\n");
			i = i+3;
			break;
	   }
   }


   printf("a = %d", i);

   return 1;
}
