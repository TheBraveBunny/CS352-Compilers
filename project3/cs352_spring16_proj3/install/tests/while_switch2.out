#functions: 1
#global vars: 1
#local vars: 1
#statements: 13
